test/ad/chainrules.jl

```diff
@@ -27,6 +27,21 @@ end
     test_frule(Bijectors.find_alpha, x, y, z)
     test_rrule(Bijectors.find_alpha, x, y, z)
 
+    if @isdefined Tapir
+        Tapir.TestUtils.test_rrule!!(
+            Xoshiro(123), Bijectors.find_alpha, x, y, z; is_primitive=true, perf_flag=:none
+        )
+        Tapir.TestUtils.test_rrule!!(
+            Xoshiro(123), Bijectors.find_alpha, x, y, 3; is_primitive=true, perf_flag=:none
+        )
+        #! format: off
```

Comment:
Is there any reason to disable the formatter here?

Reply:
I was trying to avoid it splitting all of the arguments onto different lines -- in this case it feels harder to read. I'm going to re-do this line to avoid any of these issues though. Will let you know when done.

Reply:
Okay. @devmotion I've just changed the way I've formatting this to avoid the issue.